R/tax_certainty.R

```diff
@@ -0,0 +1,33 @@
+#' Check if a taxon is certain or uncertain.
```

Comment:
Agreed - that's better. Perhaps "Classify" rather than "Summarise" though? With summarise, I'd expect an output of how many are certain and how many are uncertain.